action/protocol/staking/handlers_test.go

```diff
@@ -2365,3 +2727,31 @@ func depositGas(ctx context.Context, sm protocol.StateManager, gasFee *big.Int)
 	acc.Balance = big.NewInt(0).Sub(acc.Balance, gasFee)
 	return accountutil.StoreAccount(sm, actionCtx.Caller, acc)
 }
+
+func newconsignment(t *testing.T, bucketIdx, nonce int, senderPrivate, recipient, consignTpye, reclaim string, wrongSig bool) []byte {
+	r := require.New(t)
+	msg := action.ConsignMsgEther{
+		BucketIdx: bucketIdx,
+		Nonce:     nonce,
+		Recipient: recipient,
+		Reclaim:   reclaim,
+	}
+	b, err := json.Marshal(msg)
+	r.NoError(err)
+	h, _ := accounts.TextAndHash(b)
```

Comment:
change action.msgHash() to MsgHash, and use it here

Reply:
sure